<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>К занятию &quot;Работа с сетью. UDP клиент&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lb_sendingData">
      <property name="text">
       <string>Отправляемые данные</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lb_receiveData">
      <property name="text">
       <string>Принятые данные</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="te_sendText"/>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="te_result"/>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pb_sendDatagram">
        <property name="text">
         <string>Отправить датаграмму</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_start">
        <property name="text">
         <string>Начать передачу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_stop">
        <property name="text">
         <string>Остановить передачу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_clear">
        <property name="text">
         <string>Очистить принятые данные</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>te_result</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
